sphinx/source/docs/user_guide/quickstart.rst

```diff
@@ -167,6 +167,20 @@ which show how Bokeh can be used together with Jupyter interactive widgets:
 
 :bokeh-tree:`examples/howto/notebook_comms/Numba Image Example.ipynb`
 
+Zeppelin Notebooks
+==================
+
+At this point we should mention `Apache Zeppelin`_ notebooks.
+
+`Apache Zeppelin`_ is a web-based notebook that enables interactive data analytics.
+You can make beautiful data-driven, interactive and collaborative documents with SQL, Scala and more. It is widely
+used across the `Spark` community. Bokeh integrates seamlessly with Zeppelin
+notebooks. To view the above examples in a notebook, you would only need
+change |output_file| to |output_notebook|(notebook_type='zeppelin') instead.
```

Comment:
This will probably render better with just the double backticks for inline code, instead of the `|` which make a link substution.